experiments/rsc/package.json

```diff
@@ -39,16 +39,19 @@
     "@types/fs-extra": "^11.0.4",
     "@types/react": "^18.3.12",
     "@types/react-dom": "^18.3.1",
+    "@vitejs/plugin-react": "^4.3.4",
     "autoprefixer": "^10.4.20",
     "better-sqlite3": "^11.6.0",
     "dotenv": "^16.4.7",
     "es-module-lexer": "^1.5.4",
     "execa": "^9.5.1",
     "express": "^4.21.1",
     "fs-extra": "^11.2.0",
+    "import-meta-resolve": "^4.1.0",
     "kysely-codegen": "^0.17.0",
     "magic-string": "^0.30.14",
-    "miniflare": "^3.20241106.1",
+    "miniflare": "^3.20241205.0",
```

Comment:
Curious about the downgrade?

Reply:
@peterp its an upgrade, not a downgrade :D I think I needed it for types or exports they added related to worker options, and in the end didnt use it (yet).